### DISCUSSION TITLE
autofire keybinds

### DISCUSSION THREAD

**Discussion by Maks7594** [project64/project64#2167] [Ideas]
for people which are lazy like me, not having to use 100% power to do a mario 64 blj lol

but for real it can be used for speedruns, and just makes life easier

**Comment by IronYoshi64** [comment]
This can be done with the N-Rage Input plugin that is already included in builds of PJ64, you just need to switch to it. You can go to the Modifiers tab for Controller 1 or whatever it is, and add modifiers, Macro ones to set a button or key as a rapid fire/turbo button.
